src/signals/incident/components/form/MapSelectors/Asset/Selector_v2_removeafterfinishepic5440/DetailPanel/styled.tsx

```diff
@@ -15,7 +15,8 @@ export const StyledBackButton = styled(Button)`
 `
 
 export const StyledAssetList = styled(AssetList)`
-  margin: ${themeSpacing(2)} 0 ${themeSpacing(16)} 0;
+  margin: ${themeSpacing(2, 0, 2, 0)};
+  /* margin: ${themeSpacing(2)} 0 ${themeSpacing(16)} 0; */
```

Comment:
Do you want to keep this comment? And is the change from `2 0 16 0` to `2 0 2 0` intentional?

Reply:
change was intentional, comment removed.